src/index.html

```diff
@@ -10,7 +10,38 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="page">
+    <label>
+      <form
+        class="search-bar"
+        action="/"
+        method="post"
+        data-qa="big"
+      >
+        <input
+          class="input"
+          name="search"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+      </form>
+    </label>
+
+    <label>
+      <form
+        class="search-bar search-bar--small"
+        action="/"
+        method="post"
+        data-qa="small"
+      >
+        <input
+          class="input input--small"
+          name="search"
```

Comment:
each element should have different value in name attribute

Reply:
my input is not working without label tag also